docs/queries/filters.md

```diff
@@ -170,6 +170,23 @@ Such filter could be:
     starts before tomorrow
     ```
 
+### Created Date
+
+- `no created date`
+- `has created date`
+- `created (before|after|on) <date>`
+- `created date is invalid`
+
+When filtering queries by [start date]({{ site.baseurl }}{% link getting-started/dates.md %}#-start),
+the result will include tasks without a start date.
+This way, you can use the start date as a filter to filter out any tasks that you cannot yet work on.
```

Comment:
This bit looks like it is left over from text copied in.
I need to press on with some merging, and will need to edit the Created description anyway, as some more search features were added whilst you were working on this.
So, just to let you know that I will fix this.